code/game/objects/items/weapons/power_cells.dm

```diff
@@ -274,7 +274,7 @@
 	name = "Soteria \"Omega-Geyser 500S\""
 	desc = "Soteria Institute branded rechargeable S-standardized power cell. This one being part of the Omega line making it the be-all-end-all power cell of its type, yet to hit the open market."
 	icon_state = "meb_s_omega"
-	matter = list(MATERIAL_STEEL = 5, MATERIAL_PLASTIC = 5, MATERIAL_SILVER = 3, MATERIAL_GOLD = 1) //DEAR LOAD WHAT ARE THOSE COSTS?
+	matter = list(MATERIAL_STEEL = 1, MATERIAL_PLASTIC = 1, MATERIAL_SILVER = 1, MATERIAL_GOLD = 2)
```

Comment:
```suggestion
	matter = list(MATERIAL_STEEL = 3, MATERIAL_PLASTIC = 3, MATERIAL_SILVER = 1, MATERIAL_GOLD = 2)
```

Reply:
It use the same values at the atom & posi cells.

Reply:
Dont